pym/bob/input.py

```diff
@@ -2731,6 +2736,12 @@ def generatePackages(self, nameFormatter, envOverrides={}, sandboxEnabled=False)
                 except ParseError as e:
                     e.pushFrame(root.getPackageName())
                     raise e
+            for i in self.__aliases:
+                try:
+                    p = walkPackagePath(result, self.__aliases[i])
```

Comment:
I think it's better to expand aliases only once. Otherwise an alias could again reference an alias. But the order of the for loop is undefined.

```python
tmp = result.copy()
for i in self.__aliases:
  ..
  p = walkPackagePath(tmp, ...
```